secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: my-secret
```

Comment:
please explain

Reply:
testing though.. I just want to know if I am on point?
Just a secret.yaml file

Reply:
ok